setup.py

```diff
@@ -68,8 +68,12 @@
     # requirements files see:
     # https://packaging.python.org/en/latest/requirements.html
     install_requires=[
-        'molecule==1.25.1',
-        'ansible==2.3.2',
+        'ansible==2.4',
```

Comment:
As per the Python semantics, this will install `2.4.0.0`. Do we want `ansible==2.4.6` or `ansible<2.4`?